components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/dto/SubscriptionListDTO.java

```diff
@@ -0,0 +1,94 @@
+package org.wso2.carbon.apimgt.rest.api.store.dto;
```

Comment:
Kindly note that this is a generated code by swagger2cxf-maven-plugin (https://github.com/hevayo/swagger2cxf-maven-plugin)